doc/ref_impl/cntt-ri2/chapters/chapter03.md

```diff
@@ -14,12 +14,13 @@
 <a name="3.1"></a>
 ## 3.1 Introduction
 
-The purpose of this chapter is to list and describe the requirements on labs used to deploy and run the RI2. Overall, the RI2 serves two purposes which guide the definition of requirements in this chapter:
+The purpose of this chapter is to list and describe the requirements on labs used to deploy and run the RI2. Overall, the RI2 serves the following purposes which guide the definition of requirements in this chapter:
 
-1. RI2 serves as a baseline and proving ground for developing the [Reference Conformance 2 (RC2) specification](../../../ref_cert/RC2) as well as the [Cloud Native OVP](https://wiki.lfnetworking.org/x/agCDAQ) program.
+1. RI2 is an implementation of and a proving ground for the [Reference Architecture 2 (RA2) specification](../../../ref_arch/kubernetes). RI2 is verified against RA2 through the [Reference Conformance 2 (RC2) specification](../../../ref_cert/RC2).
+1. RI2 will align itself with the implementation work conducted within the [Cloud Native OVP](https://wiki.lfnetworking.org/x/agCDAQ) program.
 1. RI2 enables software vendors of CNFs and cloud container platforms to deploy a reference implementation in their development environment to develop and validate against.
 
-As a result of those two main use cases, RI2 must be deployable in different lab environments:
+As a result of these, RI2 must be deployable in different lab environments:
 
 1. internal development labs of software vendors, operators and 3rd party test labs,
 1. public community labs (e.g. OPNFV),
```

Comment:
if not deleted -- a precious suggestion is rejected, then correct typo
```suggestion
1. public community labs (e.g. Anuket (formerly OPNFV))
```

Reply:
@pgoyal01 I took a bit from both of the proposals. Let me know if that works